libcxx/benchmarks/stringstream.bench.cpp

```diff
@@ -1,11 +1,12 @@
 #include "benchmark/benchmark.h"
 #include "test_macros.h"
 
+#include <mutex>
 #include <sstream>
 
 TEST_NOINLINE double istream_numbers();
 
-double istream_numbers() {
+double istream_numbers(std::locale* l) {
```

Comment:
nit: please use loc over one letter names, especially l (the letter) since it's easily confused with 1 (the number).